cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go

```diff
@@ -111,6 +111,12 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 					ConfigSyncPeriod: metav1.Duration{Duration: 1},
 				},
 			}
+			zero := 0
+			obj.AuditPolicyConfiguration = kubeadm.AuditPolicyConfiguration{
+				Path:      "foo",
+				LogDir:    "/foo",
+				LogMaxAge: &zero,
```

Comment:
If the type of `LogMaxAge` is going to be changed to `int32`, then you can use `utilpointer.Int32Ptr(0)` instead :)

Reply:
sounds good!